### PR TITLE
Publisher info

### PR DESCRIPTION
**Pull request by bridiver**
## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions

### CODE REVIEW

**Comment by NejcZdovc** [comment]
@bridiver can you please update dep for native ledger as well

**Comment by bridiver** [comment]
@NejcZdovc done

File: browser/payments/BUILD.gn [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
 source_set("payments") {
-  public_configs = [
-    ":external_config"
+  public_deps = [
+    "buildflags",
   ]
 
   sources = [
+    "content_site.cc",
+    "content_site.h",
```
Comment (simonhong) [inline browser/payments/BUILD.gn]: Build failed with below log.
```
../..\brave/browser/payments/content_site.h(32,14):  error: no template named 'vector' in namespace 'std'
typedef std::vector<const ContentSite> ContentSiteList;
        ~~~~~^
../../brave/browser/payments/content_site.cc(6,10):  fatal error: 'bat/ledger/publisher_info.h' file not found
#include "bat/ledger/publisher_info.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Not sure but above two files should be moved into `brave_payments_enabled`?

Comment (bridiver) [reply browser/payments/BUILD.gn]: yep, I'll fix

Comment (bridiver) [reply browser/payments/BUILD.gn]: https://github.com/brave/brave-core/pull/294

**Pull request merged**